### DISCUSSION TITLE
Allow to specify different currencies for unit price and fee

### DISCUSSION THREAD

**Discussion by elnigno** [ghostfolio/ghostfolio#2666] [Ideas]
Taking the README example, it's possible to specify a currency for activities, but it's assumed that unit price and fee are in the same currency:

```json
{
  "activities": [
    {
      "currency": "USD",
      "dataSource": "YAHOO",
      "date": "2021-09-15T00:00:00.000Z",
      "fee": 19,
      "quantity": 5,
      "symbol": "MSFT",
      "type": "BUY",
      "unitPrice": 298.58
    }
  ]
}
```

When adding activities in the UI, it's possible to specify different currencies:

![currency](https://github.com/ghostfolio/ghostfolio/assets/2082130/c33ce4a0-ec93-49e9-ab28-ef65c35b2849)

It would be nice to be able to do the same in the UI.
```

**Comment by Sigfrodr** [comment]
This is required as many brokers don't provide fees in instrument currency, but in account currency.
So we have to manually find the correct conversion ratio at the time of the activity to convert the fee. Very time consuming.
